src/docs/primitives/a-model.md

```diff
@@ -29,3 +29,9 @@ Using scale to increase to size of the loaded tree model:
 ```html
 <a-model scale="10 10 10" src="https://aframe.io/aframe/examples/_models/tree1/tree1.dae"></a-model>
 ```
```

Comment:
perhaps we should change the table to look similar to the [`loader` component's](https://aframe.io/docs/components/loader.html)